{
    "config": {
        "abort": {
            "already_configured": "Ger\u00e4t ist bereits konfiguriert",
            "reauth_successful": "Die erneute Authentifizierung war erfolgreich"
        },
        "error": {
            "cannot_connect": "Verbindung fehlgeschlagen",
            "invalid_auth": "Ung\u00fcltige Authentifizierung",
            "unknown": "Unerwarteter Fehler"
        },
        "step": {
            "user": {
                "data": {
                    "api_key": "API-Schl\u00fcssel"
                }
            }
        }
    }
}